templates/v3/includes/_wysiwyg_editor.html

```
@@ -6,23 +6,38 @@
-  <div class="wysiwyg-editor__toolbar" x-show="jsReady" x-cloak></div>
-  <div class="wysiwyg-editor__body" x-show="jsReady" x-cloak></div>
+{% load static %}
+
+<div class="wysiwyg-v3-container">
+  <div class="wysiwyg-v3" data-wysiwyg="v3" x-data="{ jsReady: false }" x-init="
+      jsReady = true;
+      $nextTick(() => {
+        autoInit('{{textarea_id}}');
+      })"
+    >
+    <textarea
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)